.github/workflows/promote.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: canonical/charming-actions/release-charm@2.4.0
         with:
           base-channel: 22.04
-          credentials: ${{ secrets.CHARMHUB_TOKEN }}
+          credentials: ${{ secrets.CHARMCRAFT_AUTH }}
```

Comment:
Looking at the action secrets stored in this GH repo I see `CHARMHUB_TOKEN`, not `CHARMCRAFT_AUTH`, is it meant to be changed after the PR is merged?

Reply:
Yes I just updated the secret. The intent was to standardize on the secret name